app/assets/javascripts/page_refresh.js

```diff
@@ -0,0 +1,20 @@
+/***
+ * Triggers a full refreseh of the page at a specified internval.
+ *
+ * Usage:
+   = content_for :head_content do
+     = javascript_include_tag "page_refresh"
+
+   .js--pageRefresh{ data: { refresh_interval: 1.hour } }
+***/
+
+document.addEventListener("DOMContentLoaded", function() {
+  const page = document.getElementsByClassName("js--pageRefresh")[0];
+  const refreshInterval = page.dataset["refreshInterval"] || 60; // in seconds
```

Comment:
for accessible purposes, it's useful to communicate to the user beforehand that there is a timeout for this activity. https://knowbility.org/blog/2018/WCAG21-226Timeouts/

Reply:
This is a good point, but the refresh is indirectly related.  The page contains links that allow users to take action on their reservations, and the opportunity to make changes expires 12 hours after the reservation end time.  This is true whether or not the page refreshes.  I could see adding a separate ticket to make this window more transparent, but there's enough complexity there is quite a but of complexity around when reservations can be updated so I'd like to address it on its own.